modules/openshift-developer-cli-connecting-your-application-to-multiple-services-using-openshift-service-catalog.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+
+[id="connecting-your-application-to-multiple-services-using-openshift-service-catalog_{context}"]
+
+= Connecting your application to multiple services using OpenShift Service Catalog
+
+The OpenShift service catalog is an implementation of the Open Service Broker API (OSB API) for Kubernetes. This allows you to connect applications deployed in OpenShift to a variety of services. 
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* The service catalog is installed and enabled on your cluster.
+
+.Procedure
+
+* To list the services, use:
++
+----
+$ odo catalog list services
+----
+
+* To list service catalog related operations, use:
```

Comment:
This is incorrect, we are not listing anything. Perhaps you meant `To use Service Catalog related operations:`

And is Service Catalog meant to be capitalized since it's a name of a function of OpenShift?

Reply:
Checked in with the OpenShift docs team, they decided on lower case so it should be the same throughout all of the docs.